ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -765,6 +768,9 @@ impl TraceBuilder {
         if inst.is_debug_call(self.aot_mod) {
             return Ok(());
         }
+        if AOT_MOD.func(*callee).name() == "yk_trace_basicblock" {
```

Comment:
Should this be `__yk_trace_basicblock`?

Reply:
maybe, but that's how I defined it before:
https://github.com/ykjit/ykllvm/blob/main/llvm/include/llvm/Transforms/Yk/BasicBlockTracer.h#L7

Reply:
I can rename it if the naming convention is to start with `__`

Reply:
Yeah we use `__` for such things, but you can do the renaming in a separate PR after this one.

Reply:
sounds good.